ui/mainwindow.cpp

```diff
@@ -115,31 +151,23 @@ MainWindow::MainWindow(QWidget *parent)
     , ui(new Ui::MainWindow)
     , m_stateManager(this)
 {
+
+    const auto styleOverride = gammarayStyleOverride();
+    if (styleOverride) {
+        applyStyle(styleOverride);
```

Comment:
The applyStyle() hack is only needed for in-process mode, for the normal client qApp->setStyle() is a lot easier and more robust.